File: 4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        
        std::vector<int> merge{nums1};
        
        for(auto&& x : nums2)
            merge.push_back(x); 
        
        std::sort(merge.begin(), merge.end()); 
        
        if(merge.size() % 2 != 0) {
            
            int mid = merge.size()/2; 
            
            return merge[mid]; 
        }
        
        else {
            
            int mid = merge.size() / 2 - 1;
            
            return (double)(merge[mid] + merge[mid +1]) / 2;
            
            
        }
        
        
        return 0.0; 
    }
};

File: 4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        
        merge = nums1 + nums2
        
        merge.sort()
        if(len(merge) % 2 != 0):
            mid = len(merge) // 2
            return merge[mid]
        else:
            
            mid = len(merge) // 2 - 1 
            return (merge[mid] + merge[mid + 1]) / 2  
        
        
 